pontoon/base/templates/django/account/base.html

```diff
@@ -0,0 +1 @@
+../base.html
```

Comment:
All templates are related to the django-allauth, use 'socialaccount/base.html' or 'account/base.html' as their base templates. That's why I created one base template that's symlinked in two places to avoid conflicts with our base.html to avoid errors.